### ISSUE TITLE
[miio] Robot Vacuum 2 - channel values for "status#fan_power"

### ISSUE THREAD

**Issue by Stefan300381** [openhab/openhab-addons#4994]
## Expected Behavior
Binding should support values for Robot Vacuum 2 as:
Mode: Level
Mob: 105
Quiet: 101
Balanced: 102
Turbo: 103
Max: 104

## Current Behavior
values are interpreted as described under
https://github.com/marcelrv/XiaomiRobotVacuumProtocol/blob/master/FanPower.md
Mode: Level
Mob: 105
Quiet: 38
Balanced: 60
Turbo: 75
Max: 100

## Steps to Reproduce (for Bugs)
Not 100% sure how to reproduce, but I saw this in the logs:
```
Link 'statusFanPow => miio:vacuum:0704CB39:status#fan_power' has been removed.
statusFanPow changed from NULL to 75.0
```

Again, I don't think that 75 is a valid value. No clue how this was derived. 

## Your Environment
Latest stable OH 2.4
Latest stable Roborock FW 3.39_001702

**Comment by Stefan300381** [comment]
I noticed that the wrong default values (like 75 above) are retrieved once I restart openHAB.

**Comment by Stefan300381** [comment]
@davidgraeff 
this might be a stupid question as I'm not very familiar with github. What does the bug label mean? Shouldn't be someone assigned to this issue?

best regards
Stefan

**Comment by davidgraeff** [comment]
The maintainer of the binding should come by from time to time and check for issue reports. He does not get a notification on its own. If you know who that is you can reference him.

"Bug" in this context really just means this is a "defect" report and not an enhancement or infrastructure problem.

**Comment by wborn** [comment]
You can look up who's maintaining which binding in the [CODEOWNERS](https://github.com/openhab/openhab2-addons/blob/master/CODEOWNERS#L90) file.
So perhaps @marcelrv can help with this?

**Comment by marcelrv** [comment]
The new rockrobo indeed seems to have some additional values.
I'll add them to the extend they are known now.

**Comment by Stefan300381** [comment]
awesome! great community. thanks a lot

**Comment by Stefan300381** [comment]
@marcelrv stupid question, is this now fixed somehow? I'm running openHAB 2.5 M2 but I still see the same issue.

**Comment by Markkuuss** [comment]
With the Roborock S5e you can also set the mopping level.

The fan levels are almost identical to Robot Vacuum 2:

Mode: Level
Quiet: 101
Balanced: 102
Turbo: 103
Max: 104
**Soft: 105
Auto: 106**

Is it known how the mop mode is set for the Roborock S5e? Probably there's a separate channel for the 4 watering levels?
